plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/stock-status.block_theme.spec.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+import path from 'path';
+
+/**
+ * Internal dependencies
+ */
+import {
+	GeneratedPost,
+	createPostFromTemplate,
+} from '../../utils/create-dynamic-content';
+
+const TEMPLATE_PATH = path.join( __dirname, 'stock-status.handlebars' );
```

Comment:
Can we set the second param of `createPostFromTemplate` to detect relative path? like this:

```
		defaultBlockPost = await createPostFromTemplate(
			'Product Filter Stock Status Block',
			'./stock-status.handlebars',
			{}
		);
```

Reply:
we certainly can 😄 happy to add that for convenience.

Reply:
@dinhtungdu don't know what I was thinking, this doesn't make sense to do.

its because of how a file path is resolved in node.

if we pass relative path to readFile it will be determined by current working directory. cwd is based on where node was run from. 

if we pass a relative path from a test, we have no idea of the location of that test file. The test will want to resolve the filename from relative to the test file and not relative to cwd or anything else. You can pass a relative path if you want but it will be calculated from where node was run from and that could be different due to pnpm allowing you to run commands from anywhere in monorepo.

I can't right now think of a better alternative since I'd prefer to let templates go right next to tests rather than in a prescribed directory.